source/codegen/metadata/nirfmxinstr_restricted/functions.py

```diff
@@ -1197,5 +1197,22 @@
             }
         ],
         'returns': 'int32'
-    }
+    },
+	'GetSFPSessionAccessEnabled':{
```

Comment:
Nit: Can you get this new method to use the same tabbing as the entries above it? Same with the parameters and returns lines.